modules/values.yaml

```diff
@@ -1,7 +1,8 @@
 # Default global values section
 global:
-  discovery: {}
-  modulesImages: {}
+  # todo remove duplicate config values they should be in global-hooks/openapi/config-values.yaml only
+  # now we have strange behaviour in template tests
+  # probably, test helm render does not get defaults from global-hooks/openapi/config-values.yaml
```

Comment:
It sure does not, but it is about our tests code. Template tests when executed read files from various directories and merge them to get the result.

We solved the problem with template tests by moving values.yaml files to */template_tests directories for modules. Maybe there is a way to move this file somewhere too (or add the code to our tests which will read default global values from somewhere).

Reply:
I stayed it as is.